packages/admin/addon/routes/admin/roles.ts

```diff
@@ -4,5 +4,21 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
-export default class AdminRoles extends Route {}
+export default class AdminRoles extends Route {
+  /**
+   * @property { Service } store;
+   */
+  @service store: TODO;
```

Comment:
Should be able to import
`import StoreService from '@ember-data/store';`
And then
```suggestion
  @service store!: StoreService;
```